sds_data_manager/utils/stackbuilder.py

```diff
@@ -262,7 +269,7 @@ def build_backup(scope: App, env: Environment, source_account: str):
 
     """
     # This is the S3 bucket used by upload_api_lambda
-    backup_bucket_stack.BackupBucket(
+    backup_bucket_construct.BackupBucket(
         scope,
```

Comment:
For all the other constructs the scope is equal to the name of the stack. Why not this one?

Reply:
Good catch, this needs a Stack at some point, so I added one above it here to follow the same style as the others.